app/src/main/java/org/oppia/app/help/HelpActivityPresenter.kt

```diff
@@ -17,7 +17,7 @@ class HelpActivityPresenter @Inject constructor(private val activity: AppCompatA
   fun handleOnCreate() {
     activity.setContentView(R.layout.help_activity)
     setUpNavigationDrawer()
-    activity.supportFragmentManager.beginTransaction().add(
+    activity.supportFragmentManager.beginTransaction().replace(
```

Comment:
Check `StoryActivityPresenter`. In that we are first checking if the fragment is already available or not and then we are using fragment manager. So I think it would be better to write a similar code for this presenter.